agent/api/api_client.go

```diff
@@ -304,3 +307,15 @@ func (client *ApiECSClient) DiscoverPollEndpoint(containerInstanceArn string) (s
 
 	return *resp.Endpoint, nil
 }
+
+func (client *ApiECSClient) DiscoverTelemetryEndpoint(containerInstanceArn string) (string, error) {
+	resp, err := client.c.DiscoverPollEndpoint(&ecs.DiscoverPollEndpointInput{
```

Comment:
I'm not really wild about calling `DiscoverPollEndpoint` twice, but I don't think this is critical to address right now.  Let's think about refactoring this a bit in the future.